### PR TITLE
refactor: Removed weird if-else statements (DRY-er code)🥳 

### PR DESCRIPTION
**Pull request by Aradhya-Tripathi**
# Description

* Improved error handling.
* Removed redundant if-else statements (finally 🥳).

### CODE REVIEW

**Review by riju561 — APPROVED** [review]
LGTM

**Review by riju561 — APPROVED** [review]
LGTM

**Review by saxenabhishek — CHANGES_REQUESTED** [review]
this was so hard to review

File: server/githubsrm/administrator/issue_jwt.py [hunk 1]
```diff
@@ -163,13 +149,10 @@ def refresh_to_access(
...
-                ).timestamp()
-                return self.issue_key(payload=payload, get_refresh_token=True)
-            else:
-                return False
-        else:
-            return False
+        try:
+            token = self.verify_key(key=refresh_token)
+        except AuthenticationErrors:
+            return
```
[the remark this answers is not in the document]
Comment (Aradhya-Tripathi) [reply server/githubsrm/administrator/issue_jwt.py:155]: API level exception handling seemed best here as it automatically throws 401 or 403 when authentication fails, thus removing redundant checks. 🤔

File: server/githubsrm/administrator/models.py [hunk 2]
```diff
@@ -428,15 +420,9 @@ def get_all_maintainer_emails(self, project: Dict[str, Any]) -> Dict[str, Any]:
...
-        except Exception as e:
+        except Exception:
             return
-
-        doc = {}
-
-        doc["email"] = [maintainer["email"] for maintainer in maintainers]
-
-        return doc
+        return dict(email=[maintainer["email"] for maintainer in maintainers])
```
[the remark this answers is not in the document]
Comment (Aradhya-Tripathi) [reply server/githubsrm/administrator/models.py:425]: But...why? 🤔

File: server/githubsrm/administrator/issue_jwt.py [hunk 3]
```diff
@@ -35,38 +35,21 @@ def issue_key(
         payload = {**payload, **{"exp": generation_time + timedelta(minutes=expiry)}}
 
         if get_refresh_token:
-            email = (
-                payload.get("email") if payload.get("email") else payload.get("user")
+            email = payload.get("email", payload.get("user"))
+            refresh_payload = {
+                "exp": generation_time + timedelta(days=refresh_expiry),
+                "refresh": True,
+            }
```
[the remark this answers is not in the document]
Comment (Aradhya-Tripathi) [reply server/githubsrm/administrator/issue_jwt.py:42]: Makes sense lol

**Comment by Aradhya-Tripathi** [comment]
> this was so hard to review
> 

Had to get rid of weird checks everywhere 🙂

File: server/githubsrm/administrator/issue_jwt.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Aradhya-Tripathi) [reply server/githubsrm/administrator/issue_jwt.py]: refactored in 76e1217

**Review by saxenabhishek — APPROVED** [review]

**Pull request merged**